blocks/editable/format-toolbar.js

```diff
@@ -151,17 +152,24 @@ class FormatToolbar extends Component {
 						className="editable-format-toolbar__link-modal"
 						style={ linkStyle }
 						onSubmit={ this.submitLink }>
+						<label
```

Comment:
Latest JAWS version support ARIA, since a few years.
> because it simplifies the implementation by quite a bit

This is really the latest reason why one should use ARIA 🙂  and that's what I was trying to explain above.